setup.py

```diff
@@ -22,6 +22,19 @@
 this_dir = os.path.dirname(os.path.abspath(__file__))
 torch_dir = torch.__path__[0]
 
+
+def hipBLASlt_supported():
+    supported_arch = ['gfx942']
+    arch_param = os.environ.get("PYTORCH_ROCM_ARCH", None)
+    if arch_param in supported_arch:
```

Comment:
arch_param(PYTORCH_ROCM_ARCH) can be a string having many gfx archs, so this condition will fail.

Reply:
Fair point, like PYTORCH_ROCM_ARCH=gfx942,gfx90a